### PR TITLE
Add Raises section to the documentation of tf.nn.max_pool and tf.nn.max_pool2d

### PR DESCRIPTION
**Pull request by Ingkarat**
Address issue[#57978](https://github.com/tensorflow/tensorflow/issues/57978). Add Raises section to the docstring of tf.nn.max_pool and tf.nn.max_pool2d.

For max_pool, raises ValueError if
- explicit padding is used with an input tensor of rank 5.
- explicit padding is used with data_format='NCHW_VECT_C'.

For max_pool2d, raises ValueError: If explicit padding is used with data_format='NCHW_VECT_C'.

### CODE REVIEW

**Comment by Ingkarat** [comment]
For the CLA issue, it shows that there are 2 contributors possibly because the email set for git commits (non-gmail) and the email for CLA were different when creating this PR. I have changed the git email to match the CLA one but the issue still remains. I am not sure how to resolve this. Do I need to re-open the PR?

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]
Valid, but needs proper CLA.

Try `git config --global user.name "your name"` and `git config --global user.email your.email@google.com` (skip `--global` and run the above in the TF clone if you want to only do this for TF repo). Then force push to this branch

**Comment by Ingkarat** [comment]
@mihaimaruseac Thank you. I set the name and email but could not figure out how to push without making any changes and ended up creating a new empty commit. The new commit has a proper author's email and passes the CLA check, but the previous commit still fails.
Maybe I messed up somewhere. Is there a way to resolve this? Thank you again for your time.

**Comment by mihaimaruseac** [comment]
Try this

```
git switch master
git pull --rebase
git switch -
git rebase -
git rebase -i HEAD~2
```

This will open a new editor with your 2 commits. Change them to be `edit`, save and exit. Then

```
git push -f
```

If this doesn't work, it might be easier to make a new PR.

**Comment by Ingkarat** [comment]
@mihaimaruseac Thank you very much for the guidance. The CLA is now successful.

**Review by mihaimaruseac — APPROVED** [review]

**Review by rohan100jain — APPROVED** [review]

**Review by gbaned — APPROVED** [review]

File: tensorflow/python/ops/nn_ops.py [hunk 1]
```diff
@@ -4796,6 +4796,11 @@ def max_pool_v2(input, ksize, strides, padding, data_format=None, name=None):
   Returns:
     A `Tensor` of format specified by `data_format`.
     The max pooled output tensor.
+
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/nn_ops.py:4799]:
```suggestion

```

File: tensorflow/python/ops/nn_ops.py [hunk 2]
```diff
@@ -5050,6 +5055,9 @@ def max_pool2d(input, ksize, strides, padding, data_format="NHWC", name=None):
...
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/nn_ops.py:5058]:
```suggestion

```

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

File: tensorflow/python/ops/nn_ops.py [hunk 3]
```diff
@@ -4796,6 +4796,10 @@ def max_pool_v2(input, ksize, strides, padding, data_format=None, name=None):
   Returns:
     A `Tensor` of format specified by `data_format`.
     The max pooled output tensor.
+  Raises:
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/nn_ops.py:4799]:
```suggestion

  Raises:
```

File: tensorflow/python/ops/nn_ops.py [hunk 4]
```diff
@@ -5050,6 +5054,8 @@ def max_pool2d(input, ksize, strides, padding, data_format="NHWC", name=None):
...
```
Comment (mihaimaruseac) [inline tensorflow/python/ops/nn_ops.py:5057]:
```suggestion

  Raises:
```

**Review by mihaimaruseac — CHANGES_REQUESTED** [review]

**Review by mihaimaruseac — APPROVED** [review]

**Pull request merged**